crates/node/platform/src/platform.rs

```diff
@@ -429,8 +430,26 @@ impl Platform for NodePlatform {
         runtime: &Runtime,
         working_dir: &Path,
     ) -> Result<Command, ToolError> {
-        let tool = self.toolchain.get_for_version(runtime.version())?;
-        let command = actions::create_target_command(tool, context, project, task, working_dir)?;
+        let version = runtime.version();
+
+        let command = if version.is_latest() {
```

Comment:
So if it's latest, it wasn't configured?

Reply:
The version fallsback to the string "latest", so it's the only reference we have right now.